MapboxCoreMaps.podspec

```diff
@@ -22,6 +22,6 @@ Pod::Spec.new do |m|
 
   m.vendored_frameworks = 'MapboxCoreMaps.xcframework'
 
-  m.dependency 'MapboxCommon', '~> 11.0'
+  m.dependency 'MapboxCommon', '11.0.2'
```

Comment:
We have a bit stricter rule for checking MapboxCommon version that looks like:

```c++    
if (Version::getMajorVersion() != MAJOR_VERSION
     || Version::getMinorVersion() < MINOR_VERSION
     || (Version::getMinorVersion() == MINOR_VERSION && Version::getPatchVersion() < PATCH_VERSION)) {
  log & abort
}
```

@macdrevx is it possible to represent same check for spm / cocoa pods configs? I could update templates if needed.